### PR TITLE
chore: [MR-590] Bump certification version to v19

### PR DESCRIPTION
**Pull request by stiegerc**

### CODE REVIEW

File: rs/xnet/payload_builder/src/certified_slice_pool.rs [hunk 1]
```diff
@@ -352,14 +355,12 @@ struct Payload {
 }
 
 /// Mean empty payload byte size: `LabelTree` with `"streams"` and `"header"`
-/// labels; subnet ID; and serialized header (3 variable length encoded
-/// integers).
-const EMPTY_PAYLOAD_BYTES: usize = 62;
+/// labels; subnet ID.
+const EMPTY_PAYLOAD_BYTES: usize = 49;
```
Comment (stiegerc) [inline rs/xnet/payload_builder/src/certified_slice_pool.rs:359]: Just from the running the failing test, it appeared to be off by 13. Seems like an odd number, but it works.

Comment (alin-at-dfinity) [reply rs/xnet/payload_builder/src/certified_slice_pool.rs:359]: Ideally, I'd run the test with various numbers here and do a binary search for the lower and upper bound; then pick the mean of the two numbers.

But it's potentially tedious work, so I can't ask that you do it, unless you actually want to.

File: rs/xnet/payload_builder/src/certified_slice_pool.rs [hunk 2]
```diff
@@ -352,14 +355,12 @@ struct Payload {
 }
 
 /// Mean empty payload byte size: `LabelTree` with `"streams"` and `"header"`
-/// labels; subnet ID; and serialized header (3 variable length encoded
-/// integers).
-const EMPTY_PAYLOAD_BYTES: usize = 62;
+/// labels; subnet ID.
```
Comment (alin-at-dfinity) [inline rs/xnet/payload_builder/src/certified_slice_pool.rs:358]: Nit: I would explicitly mention that this leaves out the actual header blob.

```suggestion
/// Mean empty payload byte size: `LabelTree` with subnet ID, `"streams"` and
/// `"header"`labels; but excluding the header leaf.
```

File: rs/xnet/payload_builder/src/certified_slice_pool.rs [hunk 3]
```diff
@@ -352,14 +355,12 @@ struct Payload {
...
+const EMPTY_PAYLOAD_BYTES: usize = 49;
 
 /// Mean non-empty payload byte size excluding messages: `LabelTree` with
-/// `"streams"` and `"header"` labels; subnet ID; serialized header (3 variable
-/// length encoded integers); plus "messages" node.
-const NON_EMPTY_PAYLOAD_FIXED_BYTES: usize = 84;
+/// `"streams"` and `"header"` labels; subnet ID.
```
Comment (alin-at-dfinity) [inline rs/xnet/payload_builder/src/certified_slice_pool.rs:362]: Ditto:

```suggestion
/// Mean non-empty payload byte size excluding messages and header leaf: `LabelTree`
/// with subnet ID, "streams"` and `"header"` labels.
```

**Review by alin-at-dfinity — APPROVED** [review]
Nice job, thank you.

File: rs/xnet/payload_builder/src/certified_slice_pool.rs (hunk 1 above)
Comment (stiegerc) [reply rs/xnet/payload_builder/src/certified_slice_pool.rs:359]: Well I checked the output for some examples and it was always 13.

File: rs/xnet/payload_builder/src/certified_slice_pool.rs (hunk 2 above)
Comment (stiegerc) [reply rs/xnet/payload_builder/src/certified_slice_pool.rs]: Changed.

File: rs/xnet/payload_builder/src/certified_slice_pool.rs (hunk 3 above)
Comment (stiegerc) [reply rs/xnet/payload_builder/src/certified_slice_pool.rs]: Changed.

**Pull request merged**